core/exchange_test.go

```diff
@@ -169,7 +181,7 @@ func createCoreFetcher(t *testing.T, cfg *testnode.Config) (*BlockFetcher, testn
 	// flakiness with accessing account state)
 	_, err := cctx.WaitForHeightWithTimeout(2, time.Second*2) // TODO @renaynay: configure?
 	require.NoError(t, err)
-	host, port, err := net.SplitHostPort(cctx.GRPCClient.Target())
+	host, port, err := net.SplitHostPort(strings.TrimPrefix(cfg.TmConfig.RPC.GRPCListenAddress, "tcp://"))
```

Comment:
cctx.GRPCClient.Target() is the app gRPC target, which was the same as the comet one in sdk 0.46, we now explicitly use the tendermint RPC GRPC address.